### PR TITLE
10.1.0 datetimedemofix

### PR DESCRIPTION
**Pull request by Expressionless**
Fixed date time issue for Fyodor (using try/catch for a solution for the demo)

### CODE REVIEW

File: qwandaq/src/main/java/life/genny/qwandaq/utils/BaseEntityUtils.java [hunk 1]
```diff
@@ -474,33 +474,67 @@ public BaseEntity addNonLiteralAttributes(BaseEntity entity) {
...
-
-		Attribute nameAttr = new Attribute("PRI_NAME", "Name", new DataType(String.class));
-		EntityAttribute name = new EntityAttribute(entity, nameAttr, 1.0);
-		name.setValueString(entity.getName());
-		entity.addAttribute(name);
+		try {
+			updated.setValueDateTime(entity.getUpdated());
+			entity.addAttribute(updated);
+		} catch(NullPointerException e) {
+			log.error("NPE for PRI_UPDATED");
```
Comment (varunshastry) [inline qwandaq/src/main/java/life/genny/qwandaq/utils/BaseEntityUtils.java:506]: How do we figure out what was null? It'd be helpful to print the trace.

Comment (Expressionless) [reply qwandaq/src/main/java/life/genny/qwandaq/utils/BaseEntityUtils.java:506]: This is to get us across the line for the demo tomorrow. I have a more robust fix in place on another commit, this is what we agreed on for the time being.

Comment (varunshastry) [reply qwandaq/src/main/java/life/genny/qwandaq/utils/BaseEntityUtils.java:506]: Ok. Let's proceed with this PR with the understanding that the handling would be changed in subsequent PR(s).

**Review by varunshastry — APPROVED** [review]
Let's proceed with this PR with the understanding that the handling would be changed in subsequent PR(s).

**Pull request merged**